Block/Banner.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace Topsort\Integration\Block;
+
+/**
+ * Topsort Magento Extension
+ *
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
```

Comment:
I don't think these statements are true, since this projected is licensed with OGL.